lib/facilities/ppms_configuration.rb

```diff
@@ -6,14 +6,14 @@
 
 module Facilities
   class PPMSConfiguration < Common::Client::Configuration::REST
-    self.open_timeout = 1500
-    self.read_timeout = 1500
+    self.open_timeout = 60
+    self.read_timeout = 60
```

Comment:
obviously above comment applies here too 😄and read is more important to be less than `60`. Generally open is a lower value or not set at all e.g. these map to `Net::HTTP`'s timeouts and the default open_timeout is `nil` and read_timeout is `60`.

Reply:
alright, I moved the timeouts to a config file, following this guidance